integrationTest/providers.json

```diff
@@ -278,6 +278,12 @@
     "TYPE": "ROUTE53",
     "domain": "$ROUTE53_DOMAIN"
   },
+  "SAKURACLOUD": {
+    "TYPE": "SAKURACLOUD",
+    "access_token": "$SAKURACLOUD_ACCESS_TOKEN",
+    "access_token_secret": "$SAKURACLOUD_ACCESS_TOKEN_SECRET",
+    "domain": "$SAKURACLOUD_DOMAIN"
```

Comment:
**Question**: Have you sent @tlimoncelli credentials for the automated integration tests? Is that possible for Sakura Cloud? 🙏🏻

Reply:
I have not done.
I tested my own domain name on my personal account on Sakura cloud.
I will therefore consider if it is possible.

Reply:
I sent credentials for Sakura Cloud via stack overflow.